scipy/optimize/_linprog_ip.py

```diff
@@ -557,12 +558,12 @@ def _display_iter(rho_p, rho_d, rho_g, alpha, rho_mu, obj, header=False):
     # no clue why this works
     fmt = '{0:<20.13}{1:<20.13}{2:<20.13}{3:<17.13}{4:<20.13}{5:<20.13}'
     print(fmt.format(
-        rho_p,
-        rho_d,
-        rho_g,
-        alpha,
-        rho_mu,
-        obj))
+        float(rho_p),
+        float(rho_d),
+        float(rho_g),
+        float(alpha) if isinstance(alpha, numbers.Number) else alpha,
```

Comment:
From what I remember `alpha` corresponds to the size of the step taken by the solver, right? In that case, under what circumstances would it *not* be a number, unless it is ``None`` in which case wouldn't it be easier to check for ``not None`` instead?

Reply:
It's a string "-" in iteration 0. It would be easier to check for the string or not a string, but I preferred this to start the line with `float` and check for what `alpha` is rather than what it is not.

Reply:
:+1: looks good to me.